gpt4all-backend/llmodel.cpp

```diff
@@ -51,10 +51,28 @@ Dlhandle *get_implementation(std::ifstream& f, const std::string& buildVariant)
 
 static
 bool requires_avxonly() {
-#ifdef __x86_64__
-    return !__builtin_cpu_supports("avx2") && !__builtin_cpu_supports("fma");
+#ifndef _MSC_VER
+    #ifdef __x86_64__
+        return !__builtin_cpu_supports("avx2") && !__builtin_cpu_supports("fma");
+    #else
+        #ifndef __WIN32__
```

Comment:
Please just return false here and write a single comment detailing situation

Reply:
Alright, makes sense. I wanted to keep the different situation for later to be dealt with.